blueprints/automation/unifiprotect/notification_smart_motion_event.yaml

```
@@ -192,6 +192,13 @@ action:
         # Android high prority
         ttl: 0
         priority: high
+        # Android alert once (requires tag to be set)
+        alert_once: |
+          {% if tag != "" %}
+            true
+          {% else %}
+            false
+          {% endif %}
```

```suggestion
        alert_once: {{ (tag != "") | string | lower }}
```

Additionally, all of the other blueprints need the same change made.

---

Done, will update PR shortly

---

@AngellusMortis just sat down to test properly and the one-liner `alert_once: {{ (tag != "") | string | lower }}` broke the blueprint.

HA error is `Failed to load blueprint: invalid key: "{'(tag != "") | string | lower': None}"`.

The `alert_once` logic didn't need to evaluate the tag contents or casing, and [it must return boolean](https://companion.home-assistant.io/docs/notifications/notifications-basic/#alert-once). 

Were you looking to condense the logic into a one-liner?

Reverting to draft status in the meantime.